modular_skyrat/modules/fishing/code/fishing.dm

```diff
@@ -1,107 +1,144 @@
+//when someone casts their fishing rod
+#define COMSIG_START_FISHING "start_fishing"
```

Comment:
These should probably be moved to the ~skyrat defines folder since we do want defines to be available everywhere in the codebase.